### ISSUE TITLE
Getting started with Vorto

### ISSUE THREAD

**Issue by aedelmann** [eclipse/vorto#318]
A getting started with Vorto with least effort, e.g:

With Web-UI:
1) Finding an existing information model using the Repository Browser e.g. XDK
2) Generate for Web UI or CoAP

With Eclipse Toolset:
Prerequisite: Installation of Vorto IDE (Link to Installation guide)
1) Finding an existing information model using the Repository Browser e.g. XDK
2) Generate for Web UI or CoAP
3) Switch to Java perspective and see the genreated bundles

By the way link: You can use our Vorto CLI Tool as well.

**Comment by erlemantos** [comment]
I'll take this.

**Comment by andrew-from-pretoria** [comment]
I understand that the Eclipse Toolset has been deprecated. Therefore this issue is no longer relevant.
